src/command_modules/azure-cli-rdbms/azure/cli/command_modules/rdbms/commands.py

```diff
@@ -119,6 +131,24 @@ def load_command_table(self, _):
                                  setter_name='_firewall_rule_custom_setter', setter_type=rdbms_custom, setter_arg_name='parameters',
                                  custom_func_name='_firewall_rule_update_custom_func')
 
+    with self.command_group('mysql server vnet-rule', mysql_vnet_sdk) as g:
+        g.command('create', 'create_or_update')
+        g.command('delete', 'delete', confirmation=True)
```

Comment:
In the CLI, we do not prompt when delete commands only delete a single item. We prompt when deleting something that would erase multiple things (resource groups, storage accounts, database servers, etc.).  What is the rationale for prompting here?

Reply:
This can potential break the user connectivity to the server if rules are deleted by mistake , hence the additional check for confirm his/her intention.

Reply:
ok , this is taken out .